windows-gaming-driver/src/driver/qemu.rs

```diff
@@ -269,8 +269,12 @@ pub fn run(cfg: &Config, tmp: &Path, data: &Path, clientpipe_path: &Path, monito
         }
     }
 
+    if let Some(ref cmd) = cfg.additional_qemu_cmdline {
+        qemu.args(cmd.split(' '));
+    }
+
     qemu.stdin(Stdio::null());
-    trace!("qemu: {:?}", qemu);
+    debug!("qemu: {:?}", qemu);
```

Comment:
Good point. Yet, the actual full qemu invocation is very valuable information - far more relevant than any of the other output. (Just printing additional params is rather pointless - the value is straight from the config, it only debugs the yaml parser)